src/styles/components/side-panel-detail.less

```diff
@@ -226,6 +226,23 @@ Side Panel Detail
 
     width: @base-spacing-unit * 7;
 
+    &-wide {
+
+      width: @base-spacing-unit * 12;
```

Comment:
this is for the input in the side panels though, and the width is static until a certain point